pkgs/by-name/ch/check-sieve/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bison,
+  diffutils,
+  flex,
+  python3,
+  nix-update-script,
+}:
+stdenv.mkDerivation rec {
+  pname = "check-sieve";
+  version = "0.10";
+
+  src = fetchFromGitHub {
+    owner = "dburkart";
+    repo = "check-sieve";
+    rev = "check-sieve-${version}";
+    hash = "sha256-UMtkiyRGX+/lL7a+c+iZHUJhg0nb4+puSPzM5W71F9o=";
+  };
+
+  nativeBuildInputs = [
+    bison
+    flex
+    (python3.withPackages (p: [ p.setuptools ]))
```

Comment:
This should be in `nativeCheckInputs`? See https://github.com/dburkart/check-sieve/blob/master/Makefile#L33-L37

Reply:
Right